Firestore/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 - [fixed] Fixed a crash that could happen when the SDK encountered invalid
   data during garbage collection (#6721).
+- [fixed] Integer precision loss warnings on some platforms.
```

Comment:
Optional: this change is sufficiently minor that it doesn't really warrant a changelog entry.